docs/writing-providers.md

```diff
@@ -147,6 +147,20 @@ The function `GetRegistrarCorrections()` returns
 a list of corrections to be made. These are in the form of functions
 that DNSControl can call to actually make the corrections.
 
+## Step 5a: File layout
+
+Here are the filenames and their purpose.  (This is new and most providers have not been updated to use this layout, but
+`providers/cscglobal` is a good example.)
+
+- `cscglobalProvider.go` - Initializers, capabilities, and struct/type definitions
+- `dns.go` - Get/fix DNS records: GetZoneRecords, GetDomainCorrections, GenerateDomainCorrections and related functions.
+- `registrar.go` - Get/fix registrar (nameserver) info: GetRegistrarCorrections and related functions
+- `listzones.go` - ListZones and related functions
+- `convert.go` - Converting between models.RecordConfig and the provider's native format
+- `api.go` - API primatives, often an ediface in front of a vendor-provided library
```

Comment:
Might be my non-native English knowledge, but AFAIK ediface is not a word. At least not one that is commonly used.

Reply:
Very valuable feedback!    The correct word is "facade" as in  https://en.wikipedia.org/wiki/Facade_pattern

Reply:
FYI: The correct spelling is edifice.  "edifice" is a word for "building".  The "facade of an edifice" is the "front of a building".

None of that matters. I'm closing this PR.